lib/rest-adapter.js

```diff
@@ -403,6 +411,14 @@ RestAdapter.prototype._registerMethodRouteHandlers = function(router,
     // Express 4.x only supports delete
     verb = 'delete';
   }
+
+  //For now: exclude paths with parameters like :id
+  //and handler functions with err obj i.e. (err, req,res,next)
+  if (!route.path.includes(':') && handler.length < 4) {
+    handler.methodName = sharedMethod.name;
+    handler.path = route.fullPath;
+    customRouter.registerPathAndHandlers({ verb: verb, fullPath: route.fullPath }, handler);
+  }
   router[verb](route.path, handler);
```

Comment:
Here is a clue - I added some logging to `RestRouter`. 

```
$ mocha -g "strong-remoting-rest call of constructor method should allow arguments in the query"

  strong-remoting-rest
    call of constructor method
REGISTER all /testclass
HANDLE get /testclass/
      1) should allow arguments in the query
```

Please add `debug()` logs to both `registerPathAndHandlers` and `handle` so that users can easily get logs like that without any code changes. Take a look at how `express.Router` logs debug messages and add similar logs to your new router.